latch_cli/snakemake/config/utils.py

```diff
@@ -148,11 +166,14 @@ def parse_value(t: Type, v: JSONValue):
         return v, v
 
     if isinstance(v, list):
-        assert get_origin(t) is list
+        assert get_origin(t) is list or get_origin(t) is Annotated
+        if get_origin(t) is Annotated:
+            t = get_args(t)[0]
 
         sub_type = get_args(t)[0]
         res = [parse_value(sub_type, x) for x in v]
-        return [x[0] for x in res], [x[1] for x in res]
+        val, defaults = [x[0] for x in res], [x[1] for x in res]
+        return val, None if any([x is None for x in defaults]) else defaults
```

Comment:
weird, why this change?

Reply:
Since we don't provide defaults for LatchFile/LatchDir, the default would be set to None, which would cause Flyte to error. 

It seems like another option is to set the field to None and require that the user manually set the default.